rpc-jobs/RPC-AIO.yml

```diff
@@ -129,9 +129,29 @@
               Cleanup
     triggers:
       - timed: "{CRON}"
+      - github-pull-request:
+          org-list:
+            - rcbops
+          github-hooks: true
+          trigger-phrase: '.*recheck_all.*|.*recheck_{context}.*'
```

Comment:
How long do we want to run old and new jenkins side-by-side?  If not an immediate switch-over (which I'd recommend, quotas allowing), should we change these to `recheck_cit_*`?